lib/lit/i18n_backend.rb

```diff
@@ -41,7 +41,9 @@ def reset_available_locales_cache
     # @param [Hash] data nested key-value pairs to be added as blurbs
     def store_translations(locale, data, options = {})
       super
-      store_item(locale, data) if store_items? && valid_locale?(locale)
+      ActiveRecord::Base.transaction do
+        store_item(locale, data) if store_items? && valid_locale?(locale)
+      end
```

Comment:
I don't know if it affects performance, but there is not need to open transaction when if check returns false. You can do this like that:

    if store_items? && valid_locale?(locale)
      ActiveRecord::Base.transaction do { store_item(locale, data) }
    end